src/components/RoiCalculatorModal/index.tsx

```diff
@@ -236,11 +238,11 @@ const RoiCalculatorModal: React.FC<RoiCalculatorModalProps> = ({
                 {t('Staked for')}
               </Text>
               <FullWidthButtonMenu activeIndex={stakingDuration} onItemClick={setStakingDuration} scale="sm">
-                <ButtonMenuItem variant="tertiary">{t('1D')}</ButtonMenuItem>
-                <ButtonMenuItem variant="tertiary">{t('7D')}</ButtonMenuItem>
-                <ButtonMenuItem variant="tertiary">{t('30D')}</ButtonMenuItem>
-                <ButtonMenuItem variant="tertiary">{t('1Y')}</ButtonMenuItem>
-                <ButtonMenuItem variant="tertiary">{t('5Y')}</ButtonMenuItem>
+                {DURATION.map((duration) => (
+                  <ButtonMenuItem key={duration} variant="tertiary">
+                    {t(duration)}
```

Comment:
Duration array should contain translated items otherwise translation test cannot detect missing items

Reply:
Can use Trans

Reply:
Done